src/style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-form {
+  position: relative;
+  left: 5px;
+  margin-top: 15px;
+  width: 100%;
+  top: 3px;
+}
+
+.search-form__input {
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+  padding-left: 15px;
+  padding-right: 43px;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.search-form__label {
+  position: relative;
+}
+
+.search-form__label:after {
+  content: "";
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  left: 16px;
```

Comment:
You could achieve the right result with `top: 25px` and `left: 26px` for the big input and `top: 15px` and `left: 13px` for the small one. Please, learn the design carefully.